src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderList.tsx

```diff
@@ -84,37 +86,194 @@ export const LimitOrderList: FC<LimitOrderListProps> = ({ cardProps, onBack }) =
+            <Thead>
+              <Tr>
+                <Th {...thSx}>
+                  <Text translation='limitOrder.sellBuy' />
+                </Th>
+                <Th {...thSx}>
+                  <Text translation='limitOrder.limitExecution' />
+                </Th>
+                <Th {...thSx}>
+                  <Text translation='limitOrder.statusHead' />
+                </Th>
+                <Th {...thSx} width='48px'></Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {openLimitOrders.map(({ accountId, sellAssetId, buyAssetId, order }) => (
+                <LimitOrderCard
+                  key={order.uid}
+                  uid={order.uid}
+                  accountId={accountId}
+                  sellAmountCryptoBaseUnit={order.sellAmount}
+                  buyAmountCryptoBaseUnit={order.buyAmount}
+                  buyAssetId={buyAssetId}
+                  sellAssetId={sellAssetId}
+                  validTo={order.validTo}
+                  filledDecimalPercentage={bnOrZero(order.executedSellAmount)
+                    .div(order.sellAmount)
+                    .toNumber()}
+                  status={order.status}
+                  onCancelClick={handleCancelOrderClick}
+                  executedBuyAmountCryptoBaseUnit={order.executedBuyAmount}
+                  executedSellAmountCryptoBaseUnit={order.executedSellAmount}
+                />
+              ))}
+            </Tbody>
+          </Table>
+        </TableContainer>
+      )}
+      {!isLoading && (openLimitOrders === undefined || openLimitOrders.length === 0) && (
+        <Center h='full'>
+          <Text color='text.subtle' translation='limitOrder.noOpenOrders' />
+        </Center>
+      )}
+    </>
+  )
+}
+
+const HistoricalLimitOrders: FC<{
+  cardProps?: CardProps
+}> = ({ cardProps }) => {
+  const { data: ordersResponse, isLoading } = useGetLimitOrdersQuery()
+  const headBackground = useColorModeValue('gray.50', '#181c1e')
+
+  const thSx: TableColumnHeaderProps = useMemo(
+    () => ({
+      position: 'sticky',
+      top: 0,
+      zIndex: 'docked',
+      color: 'text.subtle',
+      background: headBackground,
+    }),
+    [headBackground],
   )
 
+  const historicalLimitOrders = useMemo(() => {
+    if (!ordersResponse) return []
+
+    return ordersResponse
+      .filter(({ order }) => {
+        return (
+          order.class === OrderClass.LIMIT &&
+          order.signingScheme === SigningScheme.EIP712 &&
+          ![OrderStatus.OPEN, OrderStatus.PRESIGNATURE_PENDING].includes(order.status)
+        )
+      })
+      .map(({ accountId, order }) => {
+        const { chainId } = fromAccountId(accountId)
+        const sellAssetId = cowSwapTokenToAssetId(chainId, order.sellToken)
+        const buyAssetId = cowSwapTokenToAssetId(chainId, order.buyToken)
+        return { accountId, sellAssetId, buyAssetId, order }
+      })
+  }, [ordersResponse])
```

Comment:
q: are we planning a follow up to avoid blocking the main thread with this memo?